qemu/tests/virtio_different_fs_as_shared_dir_on_host.py

```diff
@@ -0,0 +1,262 @@
+import os
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_disk
+from virttest import utils_misc
+from virttest import utils_test
+
+from virttest.utils_windows import virtio_win
+
+
+@error_context.context_aware
+def run(test, params, env):
```

Comment:
I think it's better to move the script to virtio_fs_share_data.py
Because the only difference is to setup local ext3/ext4 steps ,it's kind of setup_local_nfs.
https://github.com/autotest/tp-qemu/blob/530e547b797f52ddb02c1e9a2e66a4297f56dbce/qemu/tests/virtio_fs_share_data.py#L155
What do you think? @YongxueHong

Reply:
Hi @xiagao 
Yes, it makes sense.

Reply:
> Hi @xiagao Yes, it makes sense.

@xiagao @YongxueHong  After I check both virtio_fs_share_data  and virtio_fs_share_data_multi_backend, should this script move to virtio_fs_share_data_multi_backend instead of virtio_fs_share_data?? What's your opinion?

Reply:
> > Hi @xiagao Yes, it makes sense.
> 
> @xiagao @YongxueHong After I check both virtio_fs_share_data and virtio_fs_share_data_multi_backend, should this script move to virtio_fs_share_data_multi_backend instead of virtio_fs_share_data?? What's your opinion?

Hi @nickzhq 
Good point, it looks good to me.
Thanks.

Reply:
yes, it is better to use virtio_fs_share_data_multi_backend, good point.